iop/fs/bdmfs_fatfs/src/ff.c

```diff
@@ -3387,8 +3392,8 @@ static FRESULT mount_volume (	/* FR_OK(0): successful, !=0: an error occurred */
 	if (SS(fs) > FF_MAX_SS || SS(fs) < FF_MIN_SS || (SS(fs) & (SS(fs) - 1))) return FR_DISK_ERR;
 #endif
 
-	/* Check for a valid file system on the device. */
-	fmt = check_fs(fs, 0);
+	/* Find an FAT volume on the drive */
```

Comment:
use "a" instead of "an"

Reply:
This typo belongs to the original code, I think it is fixed in the next release